winter.mix.js

```diff
@@ -0,0 +1,8 @@
+const mix = require('laravel-mix');
+
+mix.setPublicPath(__dirname);
+
+mix.js(
+    './assets/js/src/sentry.js',
+    './assets/js/dist/sentry.js'
```

Comment:
Let's try to enforce `assets/dist/type` | `assets/src/type` across all first party plugins instead of `assets/type/dist`

Reply:
This change has been made in [b1a7785](https://github.com/wintercms/wn-sentry-plugin/pull/3/commits/b1a77851756608b8fa72cfc6b3ea8c07e3394623)